cmd/ocitools/generate.go

```diff
@@ -168,6 +168,10 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
 	if context.IsSet("env") {
 		envs := context.StringSlice("env")
 		for _, env := range envs {
+			pair := strings.Split(env, "=")
+			if len(pair) != 2 {
+				return fmt.Errorf("incorrectly specified environment variable: %s", env)
+			}
```

Comment:
You don't need this check, since variables may contain equal signs:

    $ busybox | head -n1
    BusyBox v1.24.2 (2016-07-08 12:03:25 PDT) multi-call binary.
    $ busybox sh -c 'A=B=C; echo $A'
    B=C

The current runtime-spec only talks about [limits to the variable name][1] (and even there I think the spec is [too][2] [strict][3]).

[1]: https://github.com/opencontainers/runtime-spec/blob/c66adccf2f498857a25f0b6af9e9c509085523e4/config.md#process-configuration
[2]: https://github.com/opencontainers/runtime-spec/pull/427/commits/57db50f4cac16f76241d75bed30cc62cf6aeeb42
[3]: https://github.com/opencontainers/ocitools/pull/80#issuecomment-221924588